### ISSUE TITLE
Telnet command length limit 

### ISSUE THREAD

**Issue by OxLemonxO** [TotalFreedom/BukkitTelnet#14]
It needs some limit, as chatting with high lengths can crash anyone who sees the message.

**Comment by JeromSar** [comment]
@OxLemonxO Good idea, do you have any idea what the exact limit is for telnet clients, or should it be an arbitrary number?

**Comment by OxLemonxO** [comment]
1024 was a suggestion by _windows

**Comment by OxLemonxO** [comment]
xXCocco
1000 characters can't crash the player. We are putting a limit so commands don't crash the player, and you can still do give commands.
